internal/cli/atlas/integrations/create/description.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package create
+
+const (
+	short    = "Create operations"
+	newRelic = "Create|Update New Relic integration"
```

Comment:
please be human friendly, 
```suggestion
	CreateNewRelic = "Create or update the New Relic integration"
```

Reply:
Create is also the name of the package 😿 
<img width="507" alt="Screenshot 2020-09-03 at 16 48 05" src="https://user-images.githubusercontent.com/5663078/92130632-3f272a80-ee05-11ea-9ac0-5a2e9615fb7e.png">

Reply:
oh yeah sorry, I think I copy/pasted an old version you had... I don't mind the name just mond the content, your old name is fine